crypto/ec/ecdsa_ossl.c

```diff
@@ -106,6 +106,10 @@ int ossl_ecdsa_deterministic_sign(const unsigned char *dgst, int dlen,
         ERR_raise(ERR_LIB_EC, ERR_R_PASSED_NULL_PARAMETER);
         return 0;
     }
+    if (digestname == NULL) {
+        ERR_raise(ERR_LIB_EC, EC_R_INVALID_DIGEST);
```

Comment:
Don't we have any error with "not specified" semantics? Or add the digest name as extra info?

Reply:
The term "invalid digest" includes the case that no digest has been specified, so there is not a need for another error.